projects/core/src/store-finder/facade/store-data.service.ts

```diff
@@ -13,14 +13,27 @@ export class StoreDataService {
     6: 'Sat'
   };
 
+  /**
+   * Returns store latitude
+   * @param location store location
+   */
   getStoreLatitude(location: any): number {
```

Comment:
can we create an interface for `location`?

Reply:
below functions also have `location` which would benefit from this interface

Reply:
Done, I added such interface.